### ISSUE TITLE
How to prevent adding hash # to url

### ISSUE THREAD

**Issue by koekaverna** [gilbitron/laravel-vue-pagination#29]
After click page button _url_ change to _url#_.
How to prevent this?

**Comment by gilbitron** [comment]
It sounds like this does not have anything to do with the `laravel-vue-pagination` package and is outside the scope of support offered here.

**Comment by koekaverna** [comment]
I use `laravel-vue-pagination`, vuex and `JiriChara/vuex-crud`

html:
```
<pagination :data="ideasState" @pagination-change-page="changePage"></pagination>
```

js:
```
  import { mapGetters, mapActions, mapState } from 'vuex'
  import Pagination from 'laravel-vue-pagination'

  export default {
    components: {
      IdeaCard,
      Pagination
    },
...
    computed: {
      ...mapState('ideas', {
        ideasState: state => state
      })
    },
...
    methods: {
      async changePage(page) {
        this.$router.push({name: 'ideas.sort.page', params: {order: 'top', page}})
      }
    },
  }
```

**Comment by gilbitron** [comment]
Fixed in v2.0.1 by https://github.com/gilbitron/laravel-vue-pagination/commit/f4783acb4fb635d6d2a80c6ff3435fa4835aeda6
